src/Composer/Handler.php

```diff
@@ -56,7 +56,7 @@ class Handler {
   /**
    * Command to use to call the docker compose plugin.
    *
-   * When it is V1 it thould be "docker-compose", for V2 is "docker compose".
+   * When it is V1 it thould be "compose", for V2 is "docker compose".
```

Comment:
This was correct, it referes to the command (docker-compose in v1 or docker compose in v2). I guess it was changes by search and replace.

Reply:
```suggestion
   * When it is V1 it thould be "docker-compose", for V2 is "docker compose".
```

That's correct I've didn't notice this.